recipes/psycopg2/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
```

Comment:
Leaving this here as I will try to tackle this as soon as the feedstock is created.